raw_data_QA/get_data.sh

```diff
@@ -0,0 +1,36 @@
+#Remove old data directory if it exists and make new one
+OUTPUT_DIR="data/"
+rm -r $OUTPUT_DIR
+mkdir $OUTPUT_DIR
+
+
+#Extract git hash of current prod version on website
+#Get json file from current production site
+WEBSITE_PROD="https://raw.githubusercontent.com/covid-projections/covid-projections/master/src/assets/data/data_url.json"
+PROD_VERSION_FILE="version.json"
+PROD_FILE=$OUTPUT_DIR"prod.version"
+HASH_FILE=$OUTPUT_DIR"hashinfo.txt"
+curl $WEBSITE_PROD > $PROD_FILE
+json_prod=$(grep -o 'http[^"]*' $PROD_FILE)$PROD_VERSION_FILE
+curl $json_prod > $HASH_FILE
+COVID_DATA_PUBLIC_MASTER_PROD=$(python get_hash.py $HASH_FILE)
+
+#The current prod site does not have the counties csv counties cached so we will use old ones here
+COVID_DATA_PUBLIC_MASTER_OLD="6f24dcb9a6b2203572c7509e506182be62117a38"
```

Comment:
does your use of these scripts involve updating these commit hashes? 

Add something in your readme about how to track down good commit hashes for comparison?

Reply:
Ok, I reformatted all of these (no more shell scripts 😀). Lmk if you still have thoughts. I agree for v2 it would be good to pull this into libs.datasets